pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
+
+- Backup a folder via ftps to a remote machine, encrypting with a password:
+
+`FTP_PASSWORD={{ftp_login_password}} PASSPHRASE={{encryption_password}} duplicity {{path/to/source/directory}} {{ftps://user@remotehost.tld/target/directory/path/}}`
+
+- Backup a folder to Amazon S3, doing a full backup every month:
+
+`duplicity --full-if-older-than 1M --use-new-style s3://{{bucket_name[/prefix]}}`
+
+- Delete versions older than 1 year from a backup stored on a webdav share:
+
+`FTP_PASSWORD={{webdav_login_password}} duplicity remove-older-than 1Y --force {{webdav[s]://user@remotehost.tld[:port]/some_dir}}`
+
+- List the files in a backup stored on a remote machine via ssh:
+
+`duplicity list-current-files --time {{timespec}} scp://{{user@domain.tld}}/path/to/backup/dir`
```

Comment:
Found it hiding in the man page. Apparently the timespecs are weird and specific - good thing I checked! They have to be like this:

 - `2002-01-25T07:00:00+02:00`
 - `now`
 - time in seconds since 1970 (i.e. the unix epoch)
 - `1y53m2h6m` (or any combination thereof) - specifies the time in the past to which you want to travel
 - `yyyy-mm-dd`

I'm not sure how to represent that